src/transport/websocket.ts

```diff
@@ -430,22 +429,22 @@ export default class WebSocketTransport implements SubscriptionTransport {
   private onEventMessage(
     eventMessage: Message,
     subscriptionListeners: SubscriptionListeners,
-  ): Error | void {
```

Comment:
This type annotation was probably useful (although `Error?` might be a better expression of the return type!)

Reply:
I figured the return type was misleading because all `onEventMessage` does (now!) is call into either the `subscriptionListeners`'s `onError` or `onEvent`